lib/compiler/node/index.ts

```diff
@@ -3,13 +3,26 @@ import { promises as fs } from 'fs';
 import Go from "./wasm_exec.js";
 import { fileURLToPath } from "url";
 
+const sharedFunctions = Object.create(null);
+sharedFunctions.stylePreprocess = (value: string) => value;
+
+export const init = async (options: any) => {
+  for (const key of Object.keys(sharedFunctions)) {
+    if (typeof options[key] !== 'undefined') {
+      sharedFunctions[key] = options[key];
+    }
+  }
+
+  return ensureServiceIsRunning();
+}
+
```

Comment:
Adds an `init` option that can provide WASM with any callbacks we might need. Just `stylePreprocess` for now.